runner/AndroidTestOrchestratorSample/build.gradle

```diff
@@ -26,9 +26,9 @@ ext {
     buildToolsVersion = "30.0.2"
     androidxAnnotationVersion = "1.1.0"
     guavaVersion = "29.0-android"
-    coreVersion = "1.4.0-alpha04"
-    extJUnitVersion = "1.1.3-alpha04"
-    runnerVersion = "1.4.0-alpha04"
-    rulesVersion = "1.4.0-alpha04"
-    espressoVersion = "3.4.0-alpha04"
+    coreVersion = "1.4.0-alpha05"
```

Comment:
Versions are updated in https://github.com/android/testing-samples/pull/373. (Spent some time fixing my GitHub authentication issue..) Sorry for the delay on the PR.

Reply:
> Versions are updated in #373. (Spent some time fixing my GitHub authentication issue..) Sorry for the delay on the PR.